Assets/Scripts/PlayerControls.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerControls : MonoBehaviour
+{
+    public float Speed = 10f;
+    public float JumpingPower = 10f;
+
+    private float horizontal;
+    private bool isFacingRight = true;
+
+
+    [SerializeField] private Rigidbody2D rb;
```

Comment:
If this script is attached to the same object that has the rigidbody. This should be gotten with GetComponent in the Start method.